packages/data-stores/src/onboard/actions.ts

```diff
@@ -126,7 +126,6 @@ export function* createSenseiSite( {
 			} ),
 			use_patterns: true,
 			selected_features: selectedFeatures,
-			wpcom_public_coming_soon: 1,
```

Comment:
If `wpcom_public_coming_soon: 1` is set then the website shows "Coming Soon" message to the public. Since the Launchpad displays the preview of your website in an `iframe`, the user is not logged into it. Which means it shows how it would look like for the public, not for the user.

So, basically it looks like the following.

### With `wpcom_public_coming_soon: 1`